models/annotations_models/train_random_forest_classifier_model.ipynb

```diff
@@ -1062,72 +985,1186 @@
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>r</th>\n",
+       "      <th>g</th>\n",
+       "      <th>b</th>\n",
+       "      <th>i</th>\n",
+       "      <th>ndvi</th>\n",
+       "      <th>ndwi</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>231</th>\n",
+       "      <td>646.0</td>\n",
+       "      <td>458.0</td>\n",
+       "      <td>358.0</td>\n",
+       "      <td>273.0</td>\n",
+       "      <td>125.0</td>\n",
+       "      <td>59.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         r      g      b      i   ndvi  ndwi\n",
+       "231  646.0  458.0  358.0  273.0  125.0  59.0"
+      ]
+     },
+     "execution_count": 86,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df[(df[\"rd_y\"]  == 463906.25) & (df[\"rd_x\"] == 117219.25)][selected_features]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 87,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>r</th>\n",
+       "      <th>g</th>\n",
+       "      <th>b</th>\n",
+       "      <th>i</th>\n",
+       "      <th>ndvi</th>\n",
+       "      <th>ndwi</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>9982</th>\n",
+       "      <td>646.0</td>\n",
+       "      <td>458.0</td>\n",
+       "      <td>358.0</td>\n",
+       "      <td>273.0</td>\n",
+       "      <td>59.0</td>\n",
+       "      <td>125.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "          r      g      b      i  ndvi   ndwi\n",
+       "9982  646.0  458.0  358.0  273.0  59.0  125.0"
+      ]
+     },
+     "execution_count": 87,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_annotated_export_data[(df_annotated_export_data[\"rd_y\"]  == 463906.25) & (df_annotated_export_data[\"rd_x\"] == 117219.25)][selected_features]"
```

Comment:
There is still a lot of random experimental code in this notebook. This notebook is intended to be easy to run for new cases. All this clutter makes that much harder.

Reply:
i want to keep this clutter to highlight that the NDWI and NDVI columns have been switched.

These lines of code give proof of it

Reply:
Fine.